content/components/selectpanel.mdx

```diff
@@ -64,58 +169,329 @@ Select panel should be anchored to its trigger.
     <img
       width="464"
       alt="Select panel with a back button, close button, Heading, subheading, five multiple choice options with three of them selected, and apply and cancel buttons."
-      src="https://user-images.githubusercontent.com/40274682/228910121-cfcac62f-c672-48ba-97e4-c851e1f65c49.png"
+      src="https://github.com/primer/design/assets/980622/71915284-9deb-4acf-9f41-bd2e6f8a58e3"
+    />
+    <Caption>
+      For multi selection have the user confirm their selection unless an <a href="#cancelsubmit">exception is made</a>.
+    </Caption>
+  </Do>
+  <Dont>
+    <img width="464" alt="" src="https://github.com/primer/design/assets/980622/2d783336-d879-4580-8f1b-bb9326980671" />
+    <Caption>
+      Don't force users to submit their multi-selection by closing the dialog or auto-updating their choices upon
+      selection unless an <a href="#cancelsubmit">exception is made</a>.
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+### Click-to-dismiss behavior
+
+<DoDontContainer>
+  <Do>
+    <img
+      width="464"
+      alt="Mouse cursor indicating a clicking action on the apply button."
+      src="https://github.com/primer/design/assets/980622/abea97f6-9a67-4ed7-bb52-f19e8c570902"
     />
-    <Caption>If there are more than single options in a menu, have a user confirm their choices.</Caption>
+    <Caption>
+      Guide users to save or cancel their selections to dismiss select panel to prevent confusion and data loss.
+    </Caption>
   </Do>
   <Dont>
     <img
       width="464"
-      alt=""
-      src="https://user-images.githubusercontent.com/40274682/228910122-8e455131-a126-49b0-aa31-81ac507eaf81.png"
+      alt="Mouse cursor indicating a click action outside of the dialog."
+      src="https://github.com/primer/design/assets/980622/758116fc-0d39-4822-97bb-0b670bea7044"
     />
-    <Caption>Never force users to submit their multi-select options by closing the dialog or auto-updating their choices upon selection.</Caption>
+    <Caption>
+      Don’t allow a click-outside-to-dismiss behavior for select panel. Users may be used to closing and submitting the
+      information. Doing so would result in a loss of their changes.{' '}
+    </Caption>
   </Dont>
 </DoDontContainer>
 
-### Click-to-dismiss behavior
+### Search/Filtering
+
+Including a search or filter input field not only accelerates the selection process for users but also prevents the need to burden the browser or API endpoints with returning extensive lists of items.
+
+<DoDontContainer>
+  <Do>
+    <img
+      width="464"
+      alt="Mouse cursor indicating a clicking action on the apply button."
+      src="https://github.com/primer/design/assets/980622/babe1e82-c840-4321-a8a1-fcb9e74c335e"
+    />
+    <Caption>Always keep the search/filter input in the sticky header.</Caption>
```

Comment:
Is this another change? I feel like the original API had it in the header for an accessibility reason. Apologies if this has already gone through review and I have outdated information!